xmppserver/src/main/webapp/offline-messages.jsp

```diff
@@ -296,6 +308,67 @@
         </tbody>
         </table>
     </div>
+    <br/>
+    <div class="jive-contentBoxHeader">
+        <fmt:message key="offline.messages.clean.title" />
+    </div>
+    <div class="jive-contentBox">
+        <table cellpadding="3" cellspacing="0" border="0">
+            <tbody>
+                <tr valign="top">
+                    <td colspan="2" style="padding-top: 2em;">
+                        <fmt:message key="offline.messages.clean.description" />
+                    </td>
+                </tr>
+                <tr valign="top">
+                    <td width="1%" nowrap>
+                        <b>
+                            <label for="offlinecleaner">
+                                <fmt:message key="offline.messages.clean.label" />
+                            </label>
+                        </b>
+                    </td>
+                    <td width="99%">
+                        <input type="checkbox" name="offlinecleaner" id="offlinecleaner" size="5" <%= offlinecleaner.equalsIgnoreCase("true") ? "checked" : "" %> >
+                    </td>
+                </tr>
+                <tr valign="top">
+                    <td colspan="2" style="padding-top: 2em;">
+                        <fmt:message key="offline.messages.clean.timer.description" />
+                    </td>
+                </tr>
+                <tr valign="top">
+                    <td width="1%" nowrap>
+                        <b>
+                            <label for="offlinechecktimer">
+                                <fmt:message key="offline.messages.clean.timer.label" />
+                            </label>
+                        </b>
+                    </td>
+                    <td width="99%">
+                        <input type="text" name="offlinechecktimer" id="offlinechecktimer" size="5" value="<%=offlinechecktimer%>">
+                    </td>
+                </tr>
+                <tr valign="top">
+                    <td colspan="2" style="padding-top: 2em;">
+                        <fmt:message key="offline.messages.clean.daystolive.description" />
+                    </td>
+                </tr>
+                <tr valign="top">
+                    <td width="1%" nowrap>
+                        <b>
+                            <label for="daystolive">
+                                <fmt:message key="offline.messages.clean.daystolive.label" />
+                            </label>
+                        </b>
+                    </td>
+                    <td width="99%">
+                        <input type="text" name="daystolive" id="daystolive" size="5" value="<%=daystolive%>">
```

Comment:
Please use JSTL instead of `<%=` - even if it is only a number, it's good practice to not inject raw data in the output that's displayed in the browser.

Reply:
now using JSTL style